test/ripper_parser/sexp_handlers/methods_test.rb

```diff
@@ -521,57 +498,57 @@
     describe "for yield" do
       it "works with no arguments and no parentheses" do
         _("yield")
-          .must_be_parsed_as s(:yield)
+          .must_be_parsed_in_method_as s(:yield)
```

Comment:
Let's see if Ripper will start supporting this again or if there's a setting. Prism parses this just fine.